### ISSUE TITLE
[AMP Validator] `params` missing in the output

### ISSUE THREAD

**Issue by anbarasiu** [ampproject/amphtml#34577]
### Description

After the upgrade to the latest version (1.0.35) of `amphtml-validator` node package, the validator's error output is missing the `params` array and is `undefined`.

Below's a sample output we were getting in the previous version(1.0.34). We're reliant on the `params` for some pre-processing to the final template, wherein we add the missing extensions.
```
{
    severity: 'ERROR',
    line: 289,
    col: 9,
    message: "The tag 'script' requires including the 'amp-bind' extension JavaScript.",
    specUrl: 'https://amp.dev/documentation/components/amp-bind/',
    code: 83,
    params: ['amp-state', 'amp-bind']
}
```

### Reproduction Steps

This is how we're calling the API.

```
import { Validator } from 'amphtml-validator';
constructor(
    @Inject(AMP_VALIDATOR)
    private readonly ampValidator: Validator,
  ) {}

const result = this.ampValidator.validateString('<AMP Markup passed here>', 'AMP4EMAIL');
```

**Comment by antiphoton** [comment]
Fixed by internal change (cl/376179721).

**Comment by honeybadgerdontcare** [comment]
@anbarasiu thank you for filing this issue! It should be fixed now, but please reopen if you're still having problems.
